content/francais/python-turtle/using-trinket.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "Utilisation de Trinket"
+date: 2019-07-25T13:24:17-07:00
+weight: 1
+draft: false
+---
+
+<p style="text-align: center;"><iframe width="560" height="315" src="https://www.youtube.com/embed/_DbRac3d0lo" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe></p>
+
+Pour aider Alex à vivre dans une ruche, il doit apprendre à créer un rayon de miel ! Nous utiliserons la bibliothèque `turtle` de Python pour dessiner un rayon de miel pour Alex.
+
+<img src="../media/turtle_honeycomb.png" alt="A photo of a honeycomb" width="25%" />
+
+## Utilisation de Trinket
+
+Aujourd'hui, vous utiliserez Trinket pour apprendre à coder en Python. Voici une fenêtre interactive d'exemple que vous utiliserez aujourd'hui :
+
+<iframe src="https://trinket.io/embed/python/1363ac22be" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+Avant de commencer, comprenons d'abord comment fonctionne la fenêtre. Le fichier `main.py` contient du code ou des instructions à exécuter par l'ordinateur. En appuyant sur le bouton **run**, vous dites à l'ordinateur de réaliser les instructions dans le code. Tous les résultats de l'exécution du code seront affichés sur l'écran blanc en bas, sous l'onglet **result**. Aujourd'hui, nos résultats seront une animation d'un dessin en cours de création.
+
+Ensuite, expliquons plus en détail une partie du code qui vous a été fourni dans le fichier `main.py`.
+
+{{% notice tip %}}
+
+## Bibliothèques et instructions d'importation
+
+Regardez en haut du code. Vous verrez la ligne suivante :
+
+```
+import turtle
+```
+
+Cette instruction nous permet d'importer toutes les fonctionnalités de dessin fournies par la bibliothèque `turtle`. Une **bibliothèque** est un endroit où des fonctionnalités connexes sont regroupées pour être utilisées dans le langage Python. Par exemple, la bibliothèque `turtle` introduit une tortue artistique utile dans la fenêtre des résultats, que nous pouvons contrôler en utilisant notre code pour dessiner des images.
+
+## Commentaires
+
+La ligne grise qui commence par `#` est appelée un **commentaire**. Ce sont des notes dans le code qui aident à expliquer ce qui se passe en anglais clair. Étant ignorés par l'ordinateur, ils sont souvent utilisés pour que, lorsque d'autres développeurs travaillent sur le même code, ils aient une meilleure compréhension de ce qui se passe.
```

Comment:
This line talks about code comments in "clear english", should be changed to clear french.

"ce qui se passe en **Français** clair"